src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -46,6 +47,52 @@ namespace
 
 namespace JSONHelpers
 {
+    bool isGuid(const std::wstring& str)
```

Comment:
Nit: maybe 'isValidGuid' would be better

Reply:
You're right :)